### DISCUSSION TITLE
Where do I get the realtime URL?

### DISCUSSION THREAD

**Discussion by jvbraganca** [supabase/supabase#388] [Questions] [answered]
Hello,

I tried my `API URL` with `ws://` instead of `https://` but got problems on the handshake.

I couldn't find the REAL TIME URL on the examples nor the README file. Can someone help me, please?

Edit: I am using `@supabase/realtime-js 1.0.6`

**Comment by inian** [comment, marked as answer]
Hey @jvbraganca you can use realtime functionality via our `supabase-js` library ([docs](https://supabase.io/docs/client/subscribe))

If you want to use realtime-js, the realtime url will be `ws://yourprojectid.supabase.co/realtime/v1`

**Reply by jvbraganca** [reply]
Thank you @inian!
